pandas/core/frame.py

```diff
@@ -2121,6 +2122,13 @@ def eval(self, expr, **kwargs):
         ----------
         expr : string
             The expression string to evaluate.
+        inplace : bool
+            If the expression contains an assignment, whether to return a new
+            DataFrame or mutate the existing.
+
+            WARNING: inplace=None currently falls back to to True, but
```

Comment:
in theory we should have an ``inplace`` flag on ``.query``, though in general I don't like them, it makes it a bit more consistent with the rest of the API. what do you think?